hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -16,6 +18,8 @@
 import OpenMediaMatch.storage.interface as iface
 from OpenMediaMatch.blueprints import hashing
 
+MAX_TIME = int(time.mktime((datetime.now() + timedelta(days=365 * 100)).timetuple()))
```

Comment:
I don't love this but wanted to have a max time for `disable_until_ts`

Reply:
I set this to 100 years actually, thinking we def won't hit that 😅 but I can set to lets say 5 years (?) dynamically at runtime